src/get-subseqs.py

```diff
@@ -18,8 +18,31 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
-    print(f"and the coordinates in {args.coords}")
+    # Get things from cords file
+    text = ""
+    with open(args.fasta.name + "-coordinates", "r") as file:
```

Comment:
The coordinates come as an argument. You cannot assume that they have this file name!